lib/src/multi_channel.dart

```diff
@@ -137,7 +137,7 @@ class _MultiChannel<T> extends StreamChannelMixin<T>
     // This allows it to begin connected without having to send over an id.
     _controllers[0] = _mainController;
     _mainController.local.stream.listen(
-        (message) => _inner!.sink.add([0, message]),
+        (message) => _inner!.sink.add(<Object?>[0, message]),
```

Comment:
What was the error here?

Reply:
Analyzer only - it was saying it couldn't infer the `T`:

```
T message
Couldn't infer type parameter 'E'.

Tried to infer 'Object' for 'E' which doesn't work:
  Parameter 'element' declared as     'E'
                      but argument is 'T'.
The type 'Object' was inferred from:
  Parameter 'element' declared as     'E'
                      but argument is 'int'.

Consider passing explicit type argument(s) to the generic.

dart(could_not_infer)
```